swig/perl/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+if (ENABLE_PERL)
+	include(${SWIG_USE_FILE})
+
+	include_directories(${PERL_INCLUDE_PATH})
```

Comment:
I would expect target_include_directories here but I could be wrong.

Reply:
@mpreisler You're right, I'll fix that.